<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditCoffeeForm</class>
 <widget class="QDialog" name="AddEditCoffeeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить/Редактировать кофе</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <widget class="QWidget" name="formWidget">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="sortLabel">
       <property name="text">
        <string>Сорт</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sortLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="roastLabel">
       <property name="text">
        <string>Степень обжарки</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="roastLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="grindLabel">
       <property name="text">
        <string>Молотый/В зернах</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="grindComboBox">
       <item>
        <property name="text">
         <string>Молотый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>В зернах</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="tasteLabel">
       <property name="text">
        <string>Описание вкуса</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="tasteTextEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Цена</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="priceLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="volumeLabel">
       <property name="text">
        <string>Объем упаковки</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="volumeLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="orientation">
     <enum>Qt.Orientation.Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox.Cancel|QDialogButtonBox.Ok</set>
    </property>
   </widget>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>